src/main/java/com/limechain/runtime/hostapi/OffchainHostFunctions.java

```diff
@@ -203,7 +195,7 @@ public RuntimePointerSize extOffchainNetworkState() {
         PeerId peerId = network.getHost().getPeerId();
```

Comment:
This `network` is a static singleton dependency I've missed; must be extracted in the context.

Reply:
Fixed in 5cf793d.